frontend/src/components/table/TableHeader.js

```diff
@@ -1,27 +1,27 @@
 import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
 import classnames from 'classnames';
 import PropTypes from 'prop-types';
-
+import { setActiveSortFields } from '../../actions/TableActions';
 import { shouldRenderColumn, getSizeClass } from '../../utils/tableHelpers';
 import { getTableId } from '../../reducers/tables';
 
-export default class TableHeader extends PureComponent {
+class TableHeader extends PureComponent {
   UNSAFE_componentWillMount() {
     this.setInitialState();
```

Comment:
I'm strongly against this. Since we have everything we need in the redux store already, why not use it when instantiating the table instead ? On `FETCH_DOCUMENT_SUCCESS` we get the `orderBy` value, and immediately update the table. So we can move this logic there and this whole lifecycle handler won't be needed anymore.